assets/src/admin/settings/index.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, createRoot } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Snackbar } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import SiteTable from '../../components/SiteTable';
+import SiteModal from '../../components/SiteModal';
+import SiteSettings from '../../components/SiteSettings';
+
+/**
+ * Global variable from PHP
+ */
+const API_NAMESPACE = OneDesignSettings.restUrl + '/onedesign/v1';
+const NONCE = OneDesignSettings.restNonce;
```

Comment:
@up1512001 Let's move these repeated variables to the utils file and import those as needed. Example - https://github.com/rtCamp/OneSearch-internal/pull/33#discussion_r2400772232